Bottom: 1ab0fb28e0b9814a74ed0b8d5ed94a1db0faf062
Top:    2a7a3108c3e69ac53c4c85fa041a9d265617d61f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 17:47:58 +0000

Refresh of hid-step-add-copper-layer

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 00e61b3..c366960 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -922,7 +922,6 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
                                     (side == TOP_SIDE) ? 0                   - HACK_COPPER_THICKNESS : -HACK_BOARD_THICKNESS - HACK_COPPER_THICKNESS - HACK_MASK_THICKNESS, /* Bottom */
                                     (side == TOP_SIDE) ? HACK_MASK_THICKNESS - HACK_COPPER_THICKNESS : -HACK_BOARD_THICKNESS - HACK_COPPER_THICKNESS,                       /* Top */
 #else
-  objects = object3d_from_contours (info.poly,
                                     (side == TOP_SIDE) ? -HACK_BOARD_THICKNESS / 2 - HACK_COPPER_THICKNESS                       : HACK_BOARD_THICKNESS / 2 + HACK_COPPER_THICKNESS + HACK_MASK_THICKNESS, /* Bottom */
                                     (side == TOP_SIDE) ? -HACK_BOARD_THICKNESS / 2 - HACK_COPPER_THICKNESS - HACK_MASK_THICKNESS : HACK_BOARD_THICKNESS / 2 + HACK_COPPER_THICKNESS, /* Top */
 #endif
